fieldservice_location_builder/__manifest__.py

```diff
@@ -9,7 +9,8 @@
     'author': 'Open Source Integrators, Odoo Community Association (OCA)',
     'website': 'https://github.com/OCA/field-service',
     'depends': [
-        'fieldservice'
+        'fieldservice',
+        'fieldservice_account_analytic'
```

Comment:
@osi-scampbell Why do you need this dependency?

Reply:
@osi-scampbell The module that adds the customer_id field should provide the default value so you don't have to depend on it here.